programs/stake/src/stake_instruction.rs

```diff
@@ -640,6 +646,25 @@ mod tests {
         )
     }
 
+    fn get_active_stake(
```

Comment:
```suggestion
    fn get_active_stake_for_tests(
```

to dissuade someone from making this panicking function pub and using it where they ought not

Reply:
It is defined in `mod tests`, but sure, might as well.

Reply:
wouldn't be the first time someone decided to lift a test helper :wink: